_posts/en/newsletters/2019-04-30-newsletter.md

```diff
@@ -0,0 +1,203 @@
+---
+title: 'Bitcoin Optech Newsletter #44'
+permalink: /en/newsletters/2019/04/30/
+name: 2019-04-30-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter sees another slow news week but does contain our
+regular sections on bech32 sending support, selected questions and
+answers from the Bitcoin StackExchange, and notable changes in popular
+Bitcoin infrastructure projects.
+
+## Action items
+
+*None at the time of writing.*  Note: if the Bitcoin Core release team
+are satisfied that no blocking issues were found in the fourth [release
+candidate][0.18.0] distributed last week, they intend to tag the final
+release for version 0.18.0 around the time this newsletter is being
+published.  If that happens, we'll provide detailed release coverage in
+next week's newsletter.  But please don't wait on us if you plan to
+upgrade---everything you need to know about the new version is explained
+in its release notes, which will be published with or linked to as part
+of the various release announcements on different platforms.
+
+## News
+
+*None this week.  We hope everyone is enjoying a lovely spring, fall, or
+dry season.*
+
+## Bech32 sending support
+
+*Week 7 of 24.  Until the second anniversary of the segwit soft
+fork lock-in on 24 August 2019, the Optech Newsletter will contain this
+weekly section that provides information to help developers and
+organizations implement bech32 sending support---the ability to pay
+native segwit addresses.  This [doesn't require implementing
+segwit][bech32 easy] yourself, but it does allow the people you pay to
+access all of segwit's multiple benefits.*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+It's said that "imitation is the most sincere form of flattery."  In
+this week's section, we take a quick look at a few other systems that
+are using variations on bech32.  If you're already going to need to
+implement something that's basically bech32 for another project, it's
+probably worth your time to implement it for Bitcoin too.
+
+- **LN invoices** use the bech32 format with an extended Human-Readable
+  Part (HRP) and without bech32's normal 90-character limit.  See
+  [BOLT11][] for the full specification.  Example:
+  lnbc2500u1pvjluezpp5qqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqypqdq5xysxxatsyp3k7enxv4jsxqzpuaztrnwngzn3kdzw5hydlzf03qdgm2hdq27cqv3agm2awhz5se903vruatfhq77w3ls4evs3ch9zw97j25emudupq63nyw24cg27h2rspfj9srp
```

Comment:
These long, unbroken text chunks were causing horizontal scrolling in the browser. 
<img width="1677" alt="Screen Shot 2019-04-29 at 1 55 02 PM" src="https://user-images.githubusercontent.com/1615772/56920292-76a2fb80-6a88-11e9-93dc-64032827e3b9.png">

I submitted commit 0ea06f7 which adds a wrap css rule to handle these scenarios.

<img width="1677" alt="Screen Shot 2019-04-29 at 1 55 09 PM" src="https://user-images.githubusercontent.com/1615772/56920418-c1bd0e80-6a88-11e9-847f-5d7ca59f0098.png">